### PR TITLE
Added a few docstrings to rmats.py, following the guidelines in PEP 257: Docstring Conventions 

### PR DESCRIPTION
**Pull request by helmutcarter**
Not sure if you are taking public contributions but I've gotten so much from rMATS and it only feels right to give (at least a tiny bit) back. Everything I did had been marked with TODO: Docstring for...

Thanks!

### CODE REVIEW

File: rmats.py [hunk 1]
```diff
@@ -87,9 +87,15 @@ def doSTARMapping(args): ## do STAR mapping
...
 
 def get_args():
-    """TODO: Docstring for get_args.
-    :returns: TODO
-
+    """Supplies all the neccessary arguments to the argparse package, along with appropriate help, defaults, destinations, and choices.
+    The function itself takes no arguments.
+    Unless rMATS is called in stat mode, exits with appropriate errors if any of: sequence files, gtf, or readlength arguments are missing.
+    If an output directory and/or a temporary directory aren't supplied, exits with appropriate errors.
+    Creates output directory. Cleans user supplied bam or fastq filenames to remove newlines if they follow a space and a comma.
```
Comment (EricKutschera) [inline rmats.py:94]: > remove newlines if they follow a space and a comma

`.strip(' ,\n')` will keep removing characters from the ends of the string until reaching a char that is not in `{' ', ',', '\n'}`. The order of space comma and newline is not important: https://docs.python.org/3/library/stdtypes.html#str.strip

**Review by EricKutschera — CHANGES_REQUESTED** [review]
Thanks for addressing some of the Docstring TODOs. I left a small comment about one of the new Docstrings, but otherwise the change looks good

File: rmats.py (hunk 1 above)
Comment (helmutcarter) [reply rmats.py:94]: ah, thank you. learn something new every day!

**Comment by helmutcarter** [comment]
Not sure how to mark the requested changes as made, but I fixed it :)

**Review by EricKutschera — APPROVED** [review]

**Pull request merged**